### ISSUE TITLE
Failed deployment

### ISSUE THREAD

**Issue by itseramin** [AngaBlue/express-zod-safe#3]
Hi!

Thanks for creating the package. Please review the dependabot triggered auto-deployment script, because it fails.

**Comment by AngaBlue** [comment]
Hi @itseramin , the auto-deployment script is only run when I update the package version (and associated tags) so a dependabot PR should not trigger a package publish.  That being said, I've updated dependencies and included the dependabot patches in version `v1.1.2`.

**Comment by itseramin** [comment]
Thanks! By the way I happened to notice if I want to coerce a `param` or a `query` string property and use the validated `number` value it gives an error for me, because the `RequestHandler`'s first parameter equals to `ParamsDicitionary` which has a type of `[key: string]: string`. For a workaround I added these types to the request properties with the `&` operator so they make an intersection type as patch. Any idea what's the best approach to this problem?
